tokio-fs/src/write.rs

```diff
@@ -1,11 +1,13 @@
 use crate::{file, File};
+
+use tokio_io::AsyncWrite;
```

Comment:
Why the newline separation here (and in all the other files)?

Reply:
I believe fmt alphabetizes on a per group of imports. I think it is most helpful to first see imports from the current crate, then imports from other tokio crates, then "ecosystem" imports.

Reply:
speaking of which, I didn't fmt.

Reply:
Ah, so basically you want https://github.com/rust-lang/rustfmt/issues/2979 :p

Reply:
yep